### PR TITLE
Implement fastboot continue when omitting boot img from cdba command line

### PR DESCRIPTION
**Pull request by 0xB0D**
One thing I find handy is being able to fastboot continue.

My typical workflow is compile/edit whatever then boot the default image on the target board, populate modules on either the rootfs or on a sd card, then reboot to fastboot again.

On the subsequent reboot fastboot into the rootfs with the kernel image I've compiled.

I find this a handy way to always have a default recovery image on the board I can boot.

This series makes the boot.img non-mandatory when invoking cdba.
In this case cdba will issue a new MSG_FASTBOOT_CONTINUE command to cdba-server.

Unsurprisingly MSG_FASTBOOT_CONTINUE => "fastboot continue" on the cdba-server side.

This still works ie "fastboot download; fastboot boot"
cdba -b db410c -h benchtop-cdba ./qlt-kernel/build/square_5.x-tracking/boot_t2a.img

If you drop the boot image though the flow is "fastboot continue"
cdba -b db410c -h benchtop-cdba

My test system
devices:
  - board: db410c
    qcomlt_debug_board: /dev/serial/by-id/usb-Linaro_DebugBoard_68cb7c6b75a4451502020223236460ff-if00
    console: /dev/serial/by-id/usb-Linaro_DebugBoard_68cb7c6b75a4451502020223236460ff-if02
    name: "db410c"
    fastboot: 1dcd2e70
    fastboot_set_active: true
    fastboot_key_timeout: 5

### CODE REVIEW

File: cdba.c [hunk 1]
```diff
@@ -340,6 +341,24 @@ static void request_power_off(void)
...
 
+static void request_fastboot_continue_fn(struct work *work, int ssh_stdin)
+{
+	int ret;
+
+	ret = cdba_send(ssh_stdin, MSG_FASTBOOT_CONTINUE);
+	if (ret < 0)
+		err(1, "failed to send fastboot continue request");
+	else
+		printf("Sent MSG_FASTBOOT_BOOT\n");
```
Comment (0xB0D) [inline cdba.c:352]: nope

**Comment by lumag** [comment]
@0xB0D just for the reference, meta-qcom's `initramfs-rootfs-image` includes hook to copy modules to rootfs. You might want to use it instead.

**Comment by 0xB0D** [comment]
@lumag it is either/or

https://github.com/linux-msm/cdba/pull/66/commits/8bf824b31b833ba3fb14a12e2728c0fca5b14a7c#diff-e2595c859cae197be14afcd161104e8383205cf6b64952b13114f0e6126f498fR552

**Review by lumag — APPROVED** [review]

**Comment by lumag** [comment]
@0xB0D "The base branch requires all commits to be signed." :-(

**Comment by 0xB0D** [comment]
@lumag sure I'll repost

**Comment by 0xB0D** [comment]
@lumag signed

**Pull request merged**